upstash/manifest.json

```
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fe1f17e3-11a4-4e44-b819-8781ebcc86f8",
+  "app_id": "upstash",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Visualize metrics like bandwith, keyspace, latency, hit/miss rate ... etc and from Upstash",
```

```suggestion
    "description": "Visualize data on bandwidth, keyspace, latency, and hit/miss rates from Upstash",
```
Suggesting a change so that this fits into the 80 character limit